env.example

```diff
@@ -68,7 +68,7 @@ MOCK_ELASTIC=
 ELASTIC_URL=http://127.0.0.1
 ELASTIC_PORT=9200
 # Max number of results per query
-ELASTIC_MAX_RESULTS=50
+ELASTIC_MAX_RESULTS_PER_PAGE=10
```

Comment:
This can drop to 5, since the infinite scroll will mean you don't notice not having tons of results.